pom.xml

```diff
@@ -287,6 +288,18 @@
         <version>${project.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.quarkiverse.wiremock</groupId>
+        <artifactId>quarkus-wiremock</artifactId>
+        <version>${quarkus.wiremock.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.quarkiverse.wiremock</groupId>
+        <artifactId>quarkus-wiremock-test</artifactId>
+        <version>${quarkus.wiremock.version}</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
The `scope` should not be defined in `dependencyManagement` of the parent POM, but in the module's `dependencies` instead. That makes it more explicit, as `scope` is defined where these dependencies are actually used. It's also more consistent with how it's done for other dependencies already.